pages/maker/apps/index.tsx

```diff
@@ -40,10 +40,10 @@ const AppDirectoryPage: AppDirectoryPage = ({ signIn, handleSignIn }) => {
   }));
 
   useEffect(() => {
-    async function signInCheck() {
```

Comment:
I know this is more of a proof of concept, but lets remove this, since I'm recommending we move forward with it.